unicodetools/data/math/MathClass.txt

```diff
@@ -0,0 +1,61 @@
+# File: MathClass.txt
+# Revision: 17.0
```

Comment:
Move these files down into unicodetools/data/math/revision-17/*.txt.

At some point I can then add the older versions, parse them, and then we will be able to look at MathClass history using our usual tooling.